.github/ISSUE_TEMPLATE/issue--bug-report.md

```diff
@@ -0,0 +1,97 @@
+---
+name: 'Issue: Bug report'
+about: Let us know about an unexpected error or incorrect behavior
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+<!--
+Welcome,
+
+Thanks for opening an issue. The more information you provide, the easier it is for us to assess the problem, prioritize, assign, develop, then release a fix.
+
+The IAC/Deployment team
+
+-->
+
+### Terraform Version
+
+<!---
+We ask this to be sure you are currently running a supported terraform version from your work environment. 
+
+Run `terraform version` to show the version, and paste the result between the ``` marks below.
+
+If you are not running the latest version of Terraform we support, please try upgrading because your issue may have already been fixed.
+
+If you're not sure which versions are supported, here's a link : https://github.com/sassoftware/viya4-iac-azure#terraform to help.
```

Comment:
Suggest to upload  `iac_tooling_version` info that contains tf version and more

Reply:
Yeah that will work if they have had a successful deployment but not until. What if they are starting and things are off and they can't access the sas-iac-buildinfo? I'll leave the version for now. Seems to be a good starting point.